packages/sysutils/busybox/scripts/init

```diff
@@ -470,12 +464,12 @@ force_fsck() {
   read -t120 -n1
   # The exit status is 0 if input is available
   # The exit status is greater than 128 if the timeout is exceeded
-  if [ "$?" -ne "0" -o "$?" -gt "128" ] ; then
+  if [ "$?" -ne "0" -o "$?" -gt "128" ]; then
     echo "Repairing filesystem..."
     echo ""
     /usr/sbin/fsck -T -M -y $RUN_FSCK_DISKS
     FSCK_RET="$?"
-    if [ "$(( $FSCK_RET & 8 ))" = 8 ] ; then
+    if [ "$(( $FSCK_RET & 8 ))" = 8 ]; then
```

Comment:
Same as above, shouldn't this be:
```
     if [ $(( $FSCK_RET & 8 )) -eq 8 ]; then
```
?

And again with all the similar `$FSCK_RET & #` examples below.